source/copyright.rst

```diff
@@ -0,0 +1,10 @@
+===================================
+Copyright and Licensing Information
```

Comment:
Currently, the document only covers licensing information as far as I can tell. So I'd like to suggest to change the title and file name accordingly.